extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -114,8 +117,8 @@ public function createNewBoardModal() {
 			// skip rendering
 		}
 
-		$this->response->setVal( 'title', wfMessage( 'forum-admin-create-new-board-modal-heading' )->plain() );
-		$this->response->setVal( 'submitLabel', wfMessage( 'forum-admin-create-new-board-label' )->plain() );
+		$this->setVal( 'title', wfMessage( 'forum-admin-create-new-board-modal-heading' )->plain() );
+		$this->setVal( 'submitLabel', wfMessage( 'forum-admin-create-new-board-label' )->plain() );
 
 		$form = [
 			'inputs' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "inputs" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)